include/wlr/types/wlr_layer_shell_v1.h

```diff
@@ -49,7 +49,7 @@ struct wlr_layer_surface_v1_state {
 	struct {
 		uint32_t top, right, bottom, left;
 	} margin;
-	bool keyboard_interactive;
+	uint32_t keyboard_interactive;
```

Comment:
We can now use `enum zwlr_layer_surface_v1_keyboard_interactivity` instead here.